tests/lora/test_mixtral.py

```diff
@@ -36,18 +36,22 @@ def test_mixtral_lora(mixtral_lora_files, tp_size):
     if torch.cuda.device_count() < tp_size:
         pytest.skip(f"Not enough GPUs for tensor parallelism {tp_size}")
 
-    llm = vllm.LLM(MODEL_PATH,
-                   enable_lora=True,
-                   max_num_seqs=16,
-                   max_loras=4,
-                   distributed_executor_backend="ray",
-                   tensor_parallel_size=tp_size)
+    llm = vllm.LLM(
+        MODEL_PATH,
+        enable_lora=True,
+        max_num_seqs=16,
+        max_loras=4,
+        distributed_executor_backend="ray",
+        tensor_parallel_size=tp_size,
+        max_lora_rank=32,
```

Comment:
Had to add `max_lora_rank=32` since the model has it specified: 
https://huggingface.co/dyang415/mixtral-lora-v0/blob/main/adapter_config.json#L18